pkgs/by-name/op/openresolv/package.nix

```diff
@@ -43,6 +43,6 @@ stdenv.mkDerivation rec {
     homepage = "https://roy.marples.name/projects/openresolv";
     license = lib.licenses.bsd2;
     maintainers = [ ];
-    platforms = lib.platforms.linux ++ lib.platforms.freebsd;
+    platforms = lib.platforms.linux ++ lib.platforms.freebsd ++ lib.platforms.openbsd;
```

Comment:
Also builds on Darwin FWIW; perhaps `lib.platforms.unix` is the way to go here.

Reply:
done :3